nav2_bringup/README.md

```diff
@@ -45,12 +54,22 @@ ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py use_sim_time
 
 ### Terminal 3: Launch Nav2
 
+normal bringup
+
 ```bash
 source /opt/ros/dashing/setup.bash
 ros2 launch nav2_bringup bringup_launch.py use_sim_time:=True autostart:=True \
 map:=<full/path/to/map.yaml>
 ```
 
+manually composed bringup
+
+```bash
+source /opt/ros/dashing/setup.bash
+ros2 launch nav2_bringup composed_bringup_launch.py use_sim_time:=True autostart:=True \
```

Comment:
I think this is now a commandline argument change, not a new launch file now. 

Non-composed bringup instructions need to set it to false. 

Also, capitalize "Manually Composed Bringup" and "Normal Bringup"